engine/api/workflow/dao_node.go

```diff
@@ -156,6 +157,12 @@ func insertNodeContext(db gorp.SqlExecutor, c *sdk.WorkflowNodeContext) error {
 		sqlContext.DefaultPipelineParameters = sql.NullString{String: string(b), Valid: true}
 	}
 
+	cond, errC := json.Marshal(c.Conditions)
```

Comment:
you can use `sqlContext.Conditions, err = gorpmapping.JSONToNullString(c.Conditions)`